packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-detail.component.tsx

```diff
@@ -15,11 +15,12 @@ function formatDateTime(date) {
 interface VisitDetailComponentProps {
   visit: Visit;
   patientUuid: string;
+  toggleAll: boolean;
```

Comment:
And then let's just call this `listViewOverride` or `listViewInitial`. A component should not know about its context; i.e., VisitDetailComponent should not know anything about VisitDetailOverview.